pkg/services/grafana-apiserver/service.go

```diff
@@ -54,39 +45,19 @@ type service struct {
 	*services.BasicService
 
 	restConfig *rest.Config
-	rr         routing.RouteRegister
 
-	handler   web.Handler
 	dataPath  string
 	stopCh    chan struct{}
 	stoppedCh chan error
 }
 
-func ProvideService(cfg *setting.Cfg, rr routing.RouteRegister) (*service, error) {
+func New(dataPath string) (*service, error) {
 	s := &service{
-		rr:       rr,
-		dataPath: path.Join(cfg.DataPath, "k8s"),
+		dataPath: dataPath,
 		stopCh:   make(chan struct{}),
 	}
 
-	s.BasicService = services.NewBasicService(s.start, s.running, nil).WithName("grafana-apiserver")
-
-	s.rr.Group("/k8s", func(k8sRoute routing.RouteRegister) {
```

Comment:
Does it mean this handler was never used?

Reply:
I don't know, actually 😕 but never seems unlikely. We have a handful of POC branches lying around; I am not certain of what's been used where at this point.

Reply:
The /k8s endpoint for proxying Grafana's authenticated traffic into GrafanaAPIServer was a neat trick that we tried early on. It allows the UI, for instance, to hit K8s with Grafana authentication rather easily. Its not in production use.

For now, had to disable it in the other PR by you @zserge due to some wire-related reasons.